packages/transformer/src/IModelTransformer.ts

```diff
@@ -1994,20 +2014,36 @@ export class IModelTransformer extends IModelExportHandler {
     // It is possible and apparently occasionally sensical to delete a model without deleting its underlying element.
     // - If only the model is deleted, [[initFromExternalSourceAspects]] will have already remapped the underlying element since it still exists.
     // - If both were deleted, [[remapDeletedSourceEntities]] will find and remap the deleted element making this operation valid
+    let sql: string;
```

Comment:
NitPick for readability: Suggest to move this initialization a bit lower right before it is used as: 

- It might not be required if  targetModelId is invalid.
- Currently it is a bit distracting, with this comment above.

Reply:
Good point, changed it.